news/2020-04-07-mappers-report-march.md

```diff
@@ -0,0 +1,156 @@
+
+The Ranking Criteria. For one a blessing, for others a curse. And it’s changing all the time. This month we didn’t have many changes in the Ranking Criteria, that’s why we had a great idea. From now on we will also list ongoing discussions in the Mappers’ Report to give them a bit more of attention. The Ranking Criteria affects *all* mappers, and everyone should be aware of what is going on or what is going to happen. So let’s start right away with the changes we had this month. 
+
+### All modes
+
+- **[Applause File and Passive Hitsounds](https://osu.ppy.sh/help/wiki/Ranking_Criteria#guidelines.4):** Previously a Rule, this got now moved to the Guidelines. Applause files and passive hitsounds should use .mp3 or .ogg files. The only exception of this is when using a .wav file will result in smaller file size.
+
+- **[Generalised Mode-Specific Guidelines about Custom Hitsounds](https://osu.ppy.sh/help/wiki/Ranking_Criteria#guidelines.4):** Previously split between the General and osu!catch Ranking Criteria, this guideline exist only in the general Ranking Criteria now. The hit finish in soft and normal sample sets should not be replaced with frequently used hitsounds because these can sound obnoxious for anyone disabling beatmap hitsounds. osu!taiko has an exception of this Guideline because of own mode specific hitsound sample guideline. 
+
+### osu!taiko 
+
+- **[1/3 Pattern in Futsuu](https://osu.ppy.sh/help/wiki/Ranking_Criteria/osu!taiko#-futsuu):** A change to the Rule and Guideline regarding 1/3 pattern was made. The clause about the rest moment was moved to the Guideline. 
+
+## Open Ranking Criteria Proposals 
+
+As said previously, the Ranking Criteria is a never ending process. Everyone can suggest changes to it and there are currently a lot of proposals which need more input from the community! Every proposal can be found in the [Ranking Criteria Forum](https://osu.ppy.sh/community/forums/87). Here a small list of the most important and longest open one:
+
+For all modes:
+- **[Unofficial Cuts/Mixes of Songs must be Specific](https://osu.ppy.sh/community/forums/topics/917043)**
+- **[General RC Change regarding CS Values](https://osu.ppy.sh/community/forums/topics/1020515)**
+
+For osu!:
+- **[Proposal for Easy/Normal Specific Guideline: Spinner Approach](https://osu.ppy.sh/community/forums/topics/1010191)**
+- **[Intuitive Reverse Arrow Movement](https://osu.ppy.sh/community/forums/topics/1033709)**
+
+For osu!catch:
+- **[Dashes and Higher-Snapped Dashes on Rain difficulties](https://osu.ppy.sh/community/forums/topics/1033696)**
+
+For osu!mania:
+- **[Implementing Key-Mode Specific Ranking Criterias for every Key-Mode](https://osu.ppy.sh/community/forums/topics/1009849)**
+
+These and many others are currently discussed and need more input. So head right over to the Ranking Criteria Forum and be part of the changes which will affect the future of mapping!
+
+## <a id=”nat-meetings”></a>NAT Meetings
```

Comment:
need to be careful with these smart quotes, I don't think they'll work here

Reply:
Oh no. That's what happen when writing in google doc and copy it to github :blobsweat:
Fixed them all!